trace-reader/src/processing.rs

```diff
@@ -22,26 +22,31 @@ use supermusr_streaming_types::{
 };
 
 /// Reads the contents of trace_file and dispatches messages to the given Kafka topic.
+#[allow(clippy::too_many_arguments)]
```

Comment:
No action is needed immediately, but if you find yourself adding suppressions like this it should be taken as a sign that there might be a design/architecture issue.